### ISSUE TITLE
sqlite3 vs postgress

### ISSUE THREAD

**Issue by basplas** [activeadmin-plugins/active_admin_import#80]
Hi there,

I have a development environment with `sqlite3` and a production (Heroku) environment with `postgress`. The gem works fine for the local version, but tells me `You can import only valid csv file` when I try to import the same file into the production environment.

Here is part of my heroku logs:
```
2015-11-24T11:55:26.504300+00:00 app[web.1]: Processing by Admin::ProductsController#do_import as HTML
2015-11-24T11:55:26.504458+00:00 app[web.1]:   Parameters: {"utf8"=>"✓", "authenticity_token"=>"Nnqs3Jb/LfqrIOUvrPUT9Y4lRzh9YpaSGyFct2d19WNMYz/RzAsi59G99/LvyXXLuolAB5jgCiKDTzYvzTGa8Q==", "active_admin_import_model"=>{"file"=>#<ActionDispatch::Http::UploadedFile:0x007f6e05b1c040 @tempfile=#<Tempfile:/tmp/RackMultipart20151124-3-a4lxja.csv>, @original_filename="algemene-test-products.csv", @content_type="text/x-comma-separated-values", @headers="Content-Disposition: form-data; name=\"active_admin_import_model[file]\"; filename=\"algemene-test-products.csv\"\r\nContent-Type: text/x-comma-separated-values\r\n">}, "commit"=>"Import"}
2015-11-24T11:55:26.512911+00:00 app[web.1]:   AdminUser Load (1.8ms)  SELECT  "admin_users".* FROM "admin_users" WHERE "admin_users"."id" = $1  ORDER BY "admin_users"."id" ASC LIMIT 1  [["id", 2]]
2015-11-24T11:55:26.528978+00:00 app[web.1]: DEPRECATION WARNING: action_class is deprecated and will be removed from Formtastic 4.0 (configure Formtastic::FormBuilder.action_class_finder instead (upgrade guide on wiki: http://bit.ly/1F9QtKc )). (called from realtime at /app/vendor/ruby-2.0.0/lib/ruby/2.0.0/benchmark.rb:296)
```
Any ideas on what's going on?

Cheers,
Sebastian

**Comment by Fivell** [comment]
looks like validations are not passed. 
update and try again this https://github.com/activeadmin-plugins/active_admin_import/commit/fd0100f15eca60f9bf2149bb478712aba405bf38
I've just added new content type

**Comment by Fivell** [comment]
@basplas , up

**Comment by basplas** [comment]
Perfect, works like a charm now!

**Comment by Fivell** [comment]
fixed by #81
